test/Transactions/Orleans.Transactions.Tests/Runners/FaultInjectionTransactionTestRunner.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Orleans.Transactions.Tests.DeactivatingInjection;
+using Orleans.Transactions.Tests.DeactivationTransaction;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Orleans.Transactions.Tests
+{
+    public class FaultInjectionTransactionTestRunner : TransactionTestRunnerBase
+    {
+        public FaultInjectionTransactionTestRunner(IGrainFactory grainFactory, ITestOutputHelper output)
+         : base(grainFactory, output)
+        { }
+
+        [SkippableFact]
+        public async Task SingleGrainReadTransaction()
+        {
+            const int expected = 5;
```

Comment:
This is a nit, but I believe .NET naming conventions call for capitalizing constants.